classes/RepositoryInterface.php

```diff
@@ -111,50 +111,50 @@ public function insert(array $data);
      * @param array            $data
      * @return int
      */
-    public function update($id, array $data);
+    public function update($id, array $data): int;
 
     /**
      * @param object $entity
      * @return mixed
      */
-    public function load($entity);
+    public function load(object $entity);
```

Comment:
Same as for the EventDispatcherDecorator, the Repository class does have a object return type.